### PR TITLE
Svi backend

### PR DESCRIPTION
**Pull request by eriwarr**
## Preview Link

<a href="https://deploy-preview-1651--health-equity-tracker.netlify.app/" target="_blank">

<br />
Click to demo updates </a>  

## Description
This PR adds support for the SVI implementation on the backed in Python and sets up the data to be used on the cloud. It also includes a helper function to ensure that SVI data is filtered and formatted before being sent to the frontend. 

## Motivation and Context

These changes set up the backend so that the data can be retrieved through big query. 

## How has this been tested?

This PR includes a test file in the testing directory which ensures that the data is formatted at expected. It also includes a unit test for SVI formatting and filtering. 

## Screenshots (if appropriate):
Screen Shot 2022-07-11 at 5 14 28 PM

## Types of changes

- New feature now allows the users to see the svi at the county level on the population card.

### CODE REVIEW

File: python/datasources/cdc_svi_county.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import ingestion.standardized_columns as std_col
+
+from datasources.data_source import DataSource
+from ingestion import gcs_to_bq_util
+import numpy as np
+
+from ingestion.merge_utils import (merge_county_names)
+
+BASE_CDC_URL = 'https://data.cdc.gov/resource/8xkx-amqh.csv'
```
Comment (joshzarrabi) [inline python/datasources/cdc_svi_county.py:9]: @eriwarr I think we can delete these two lines

File: python/tests/datasources/test_cdc_svi_county.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    assert mock_csv.call_count == 1
+
+    expected_df = pd.read_csv(GOLDEN_DATA, dtype={
+        'county_fips': str,
+    })
+    assert_frame_equal(
+        mock_bq.call_args_list[0].args[0], expected_df, check_like=True)
+
+
+# @mock.patch('ingestion.gcs_to_bq_util.load_csv_as_df_from_data_dir', return_value=get_real_svi_as_df())
```
Comment (joshzarrabi) [inline python/tests/datasources/test_cdc_svi_county.py:60]: Lets remove this commented out test

File: python/tests/datasources/test_cdc_svi_county.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+from pandas._testing import assert_frame_equal
+
+from datasources.cdc_svi_county import CDCSviCounty, format_svi
+
+# insert unit tests
+
+
+def test_format_svi():
+    assert format_svi(0.4354) == .44
+    # assert format_svi(2) == ValueError
```
Comment (joshzarrabi) [inline python/tests/datasources/test_cdc_svi_county.py:14]: Do we want this line?

**Review by joshzarrabi — CHANGES_REQUESTED** [review]

**Review by joshzarrabi — APPROVED** [review]
Looks great, thank you! Can you add some more context into the pr comments and link to the SVI issue please?

**Review by benhammondmusic — APPROVED** [review]
awesome!

**Pull request merged**